app/components/page-home/page-home.component.js

```diff
@@ -5,14 +5,15 @@ const pageHomeComponent = {
   bindings: {},
   template: `
 		<md-content class="page-home">
-			<h2 class="md-display-1">Modern starter for AngularJS + Angular Material</h2>
+			<h2 data-testid="title" class="md-display-1">Modern starter for AngularJS + Angular Material</h2>
 			<img 
+			    data-testid="hero-image"
 				class="hero-img" 
 				src="https://res.cloudinary.com/fyodorio/image/upload/v1611994660/my-logos/hotrod-logo_rhkfxy.jpg" 
 				alt="ng-hotrod">
```

Comment:
I am okay with using empty alt text for the image in the top nav as mentioned in https://github.com/xlts-dev/angularjs-material-vite/pull/2#pullrequestreview-1339804950. But I think that we should add some more descriptive text here.

Reply:
Changed alt text to `Sketch of an old car with the Angular logo on the door`